### PR TITLE
Rename ambiguous Pusher.verbose to Pusher.logToConsole. Add some tests.

### PR DESCRIPTION
**Pull request by leggetter**

### CODE REVIEW

**Comment by leggetter** [comment]
@zimbatm Renamed to `Pusher.logToConsole` https://github.com/pusher/pusher-js/commit/52e10340411e03ddcabe7d94e5875c17fe1373cf

Usage will now be:

```html
<script src="//js.pusher.com/3.0/pusher.min.js"></script>
<script>
  Pusher.logToConsole = true;

  var pusher = new Pusher('APP_KEY');
</script>
```

File: README.markdown [hunk 1]
```diff
@@ -24,31 +25,48 @@ The following topics are covered:
 
 ## Installation
 
-Via the Pusher CDN (or [CDNJS](https://cdnjs.com/libraries/pusher)):
```
Comment (zimbatm) [inline README.markdown]: Was CDNJS removed on purpose ?

**Comment by zimbatm** [comment]
@leggetter thanks for pushing this through, `logToConsole` is a good compromise.

Last thing would be to restore the CDNJS reference and remove the unnecessary intermediate commits. I am happy to do it, just waiting for your approval so we don't both force-push on the branch.

**Comment by leggetter** [comment]
I don't think we should add the CDNJS reference back. 

As part of [the merge](https://github.com/pusher/pusher-js/commit/37b755fc68c38626cd7797549496bcf87b89eed8) I left this comment:

> If we are going to list CDNJS then we should push updates to it as part of the release process. Since that’s not part of the release process right now it’s being removed.

File: README.markdown (hunk 1 above)
Comment (leggetter) [inline README.markdown]: Yeah, please seem comment below.

**Comment by leggetter** [comment]
Replaced by #121

**Pull request closed without merging**